test/demo/index.html

```diff
@@ -552,6 +562,16 @@ <h3>Timeseries</h3>
         .prepare()
         .render(metricData);
 
+      window.metric = new KeenDataviz({
+          container: '#nested .chart-demo-metric-previous-results',
+          type: 'metric',
+          title: 'High-fives today',
+          results,
+          previousResults
+        })
+        .prepare()
+        .render(results);
+
```

Comment:
to remove:
:572
:573
because we already did set results in the constructor's config